resources/dicts/events/death/general.json

```diff
@@ -1731,7 +1731,7 @@
         "sub_type": [ "murder" ],
         "tags": [ "all_lives" ],
         "weight": 20,
-        "event_text": "A yowl took m_c by surprise as {PRONOUN/m_c/subject} {VERB/m_c/were/was} patrolling the territory. r_c crashed into view, meowing about how m_c had to come and see. m_c followed the distressed warrior to a small ravine. Suddenly, {PRONOUN/m_c/subject} felt a sharp shove from behind, letting out a yowl of surprise that was cut short as {PRONOUN/m_c/subject} landed on the sharp rocks below.",
+        "event_text": "m_c was shoved into a ravine by r_c, having been lured near by trickery.",
```

Comment:
Lured by trickery and deceptive words, m_c was shoved into a ravine by r_c, killing them.